Jenkinsfile

```diff
@@ -0,0 +1,40 @@
+#!groovy
+
+def nodeLabel = 'docker'
+
+/* Only keep the 10 most recent builds. */
+properties([[$class: 'BuildDiscarderProperty',
+                strategy: [$class: 'LogRotator', numToKeepStr: '10']]])
+
+node(nodeLabel) {
+  stage 'Checkout'
+  checkout scm
+
+  stage 'Build'
+
+  /* Call the maven build. */
+  mvn "clean install -B"
+
+  /* Archive the test results */
+  step([$class: 'JUnitResultArchiver', testResults: '**/target/surefire-reports/TEST-*.xml'])
+
+  /* Archive the build artifacts */
+  step([$class: 'ArtifactArchiver', artifacts: 'target/*.hpi'])
+}
+
+/* Run maven from tool "mvn" */
+void mvn(def args) {
+    /* Get jdk tool. */
+    String jdktool = tool name: "jdk8", type: 'hudson.model.JDK'
```

Comment:
I haven't been willing to build released versions of the git plugin or the git client plugin with Java 8 because I assumed that would break users running older versions.  Test builds (as executed in a pipeline) are no harm with Java 8, but I have preferred to test with the same JDK I'll use to create the release.